src/sgp/FileMan.cc

```diff
@@ -57,7 +57,7 @@ uint64_t FileMan::getFreeSpace(const ST::string& path)
 	if (!Fs_freeSpace(path.c_str(), &bytes))
 	{
 		RustPointer<char> err(getRustError());
-		STLOGW("FileMan::getFreeSpace('{}') failed: %s", path, err.get());
+		SLOGW("FileMan::getFreeSpace('{}') failed: %s", path, err.get());
```

Comment:
Looks like there is an error in the original format string. It should be replaced with `{}` too.